README.md

```diff
@@ -1 +1,218 @@
-# node-patterns-concurrency
\ No newline at end of file
+# Concurrency and Other Async Patterns
+
+## Overview
+
+Asynchronous code is more efficient than synchronous, but to write good async code you need to know at least a few Node patterns.
```

Comment:
I'm not sure what this would mean (stated so generally). The performance implications of async vs. sync code are difficult to compare, and I don't think that we want to bog students down with this discussion.

Certainly it's generally the case in Node.js that async code will have better performance characteristics than similar code executed synchronously -- but I think we should not frame the problem around performance when we're talking about error-handling.